src/hooks/index.js

```diff
@@ -202,13 +202,25 @@ exports.restrictToCurrentSchool = hook => {
 		});
 		if (access)
 			return hook;
-		hook.params.query.schoolId = res.data[0].schoolId;
+		if (hook.method == "get" || hook.method == "find" || hook.method == "remove") {
```

Comment:
Die Frage die sich mir noch stellt ist, ob `remove` Sinn ergibt?
Die wenigstens remove requests werden eine schoolId mitliefern (habe auf Anhieb keinen gefunden), heißt in den 2ten Case (Zeile 208) wird er nie reinspringen und der erste Case (Zeile 206) wird keine Auswirkung auf den remove Request haben.

Reply:
@Langleu wie stellen wir bei einem remove sicher das es ein item der eigenen schule ist?

Reply:
Ja, remove funzt, hab zwei Tests ergänzt